src/dali/plugin/input/rest/kraken.py

```diff
@@ -226,174 +196,150 @@ def _compute_tx_set(self, trade_history, ledger, processed_transactions) -> List
 
             amount: RP2Decimal = abs(RP2Decimal(record[_AMOUNT]))
             asset: str = self.base_id_to_base[record[_ASSET]]
-            kwargs: Dict[str, str] = {
-                'plugin': self.__PLUGIN_NAME,
-                'unique_id': Keyword.UNKNOWN.value,
-                'raw_data': str(record),
-                'timestamp': str(timestamp_value),
-                'spot_price': '0',
-                'asset': asset,
-                'notes': key,
-            }
+            raw_data = str(record)
 
             if record[_TYPE] == _WITHDRAWAL or record[_TYPE] == _DEPOSIT:
                 is_deposit: bool = record[_TYPE] == _DEPOSIT
                 is_withdrawal: bool = record[_TYPE] == _WITHDRAWAL
+                spot_price: str = '0'
```

Comment:
There probably is never a situation where `spot_price` would be 0. Even for a transfer, it is important to have the spot price at the time of the transfer to calculate the cost of the transaction fee. If the `spot_price` is unknown you can you use `Keyword.UNKNOWN.value` and and `pair_converter` plugin will grab it for you.